thoth/storages/ceph_cache.py

```diff
@@ -45,7 +45,10 @@ def retrieve_document_ttl(self, document_id: str) -> float:
 
         # Uses UTC time to be environment agnostic (no timezone)
         time_lived = (
-            datetime.now() - self.ceph.retrieve_document_attr(object_key=document_id, attr="LastModified")
+            datetime.now()
+            - self.ceph.retrieve_document_attr(object_key=document_id, attr="LastModified").replace(
+                tzinfo=datetime.timezone.utc
```

Comment:
str.replace format are normal in `,` format right?
should this be:
```suggestion
                tzinfo, datetime.timezone.utc
```

Reply:
Thanks for the clarification.